Frontend/src/data/rental/onSave.js

```diff
@@ -44,44 +29,65 @@ const newItemStatus = (rental) => {
   }
 };
 
-export default async (rental, closePopup, updateItemStatus, createNew) => {
-  setNumericValuesDefault0(rental, columns);
+const updateItemStatus = async (item, status) => {
+  await Database.updateDoc({ ...item, status });
+  await WoocommerceClient.updateItem(item);
+  notifier.success(
+    `'${item.name}' wurde als ${
+      item.status === "instock" ? "verfügbar" : "verliehen"
+    } markiert.`
+  );
+};
+
+export default async (context) => {
+  const { doc, closePopup, createNew, contextVars } = context;
+  setNumericValuesDefault0(doc, columns);
+
+  // item changed, reset initial item to status available
+  if (contextVars.initialItemId && contextVars.initialItemId !== doc.item_id) {
+    try {
+      const initialItem = await fetchItemById(contextVars.initialItemId);
+      await updateItemStatus(initialItem, "instock");
+      notifier.warning(
+        `Status des Gegenstandes mit ID '${contextVars.initialItemId}' wurde auf 'verfügbar' geändert. Bitter überprüfe ob das stimmt.`,
```

Comment:
Maybe we can also save something like `initicalItemName` and display it here? I think that would be more useful than just the item ID.

Reply:
You are right, I also added the name.